README.md

```diff
@@ -22,11 +22,11 @@ Run `flutter doctor` and fix everything it complains before proceeding
 
 - Copy `env.sh.example` and set it up for your machine
 - Ensure you have run `gomobile init`
-- In Android Studio, make sure you have the current ndk installed by going to Tools -> SDK Manager, go to the SDK Tools tab, check the `Show package details` box, expand the NDK section and select `27.0.12077973` version.
-- Ensure you have downloaded an ndk via android studio, this is likely not the default one and you need to check the
+- In Android Studio, make sure you have the current NDK installed by going to Tools → SDK Manager, go to the SDK Tools tab, check the `Show package details` box, expand the NDK section and select `27.0.12077973` version.
+- Ensure you have downloaded a NDK via android studio, this is likely not the default one, and you need to check the
```

Comment:
Nit: This should in fact be `an NDK`, because `N` is a vowel sound when pronounced as a single letter.